chromium_src/chrome/common/url_constants.cc

```diff
@@ -174,6 +174,13 @@ const char kPasswordManagerLearnMoreURL[] =
     "https://support.brave.com/hc/en-us/articles/360018185951-How-do-I-use-the-built-in-password-manager-";
 #endif
 
+const char kPaymentMethodsLearnMoreURL[] =
+#if defined(OS_CHROMEOS)
```

Comment:
Wasn't sure if there's any point to this conditional? It looks like we're defining the same string either way. Unless we're just trying to match the structure of the original file...

Reply:
This is intentional to match the upstream file which this is a copy of as close as possible.